src/core/stream_send.c

```diff
@@ -1069,17 +1089,28 @@ QuicStreamSendWrite(
     }
 
     if (Stream->SendFlags & QUIC_STREAM_SEND_FLAG_SEND_ABORT) {
```

Comment:
Since the logic really is completely different below, should we have a separate send flag type now?

Reply:
Adding the SEND flag I think will make things more explicit.

Reply:
Added